SmartTutor/Controllers/KnowledgeComponent/KnowledgeComponentProfile.cs

```diff
@@ -0,0 +1,13 @@
+﻿using AutoMapper;
+using SmartTutor.Controllers.KnowledgeComponent.DTOs;
+
+namespace SmartTutor.Controllers.KnowledgeComponent
+{
+    public class KnowledgeComponentProfile : Profile
+    {
+        public KnowledgeComponentProfile()
+        {
+            CreateMap<KnowledgeComponentModel.KnowledgeComponents.KnowledgeComponent, KnowledgeComponentDTO>();
```

Comment:
Remove the KnowledgeComponentModel.KnowledgeComponents and add using to the top.